tools/distrib/python/xds_protos/build.py

```diff
@@ -65,22 +61,20 @@ def localize_path(p):
 NAMESPACE_PACKAGES = ["google"]
 
 
-def add_test_import(
-    proto_package_path: str, file_name: str, service: bool = False
-):
+def add_test_import(proto_package_path: str, file_name: str, service: bool = False):
     TEST_IMPORTS.append(
         "from %s import %s\n"
         % (
-            proto_package_path.replace("/", "."),
-            file_name.replace(".proto", "_pb2"),
+            proto_package_path.replace("/", ".").replace("-", "_"),
```

Comment:
In what cases does the proto package path include a `-` character?  I don't think that should happen.

Reply:
There was some message from envoy, like `test-data` - I do not have that CI log. @gnossen wrote this change.